.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+# Problem/Motivation
+(Why the issue was filed, steps to reproduce the problem, etc.)
+
+## Proposed resolution
+(Description of the proposed solution, the rationale behind it, and workarounds for people who cannot use the patch.)
+
+## Repeatable
+Always|Sometimes|Specific conditions|Specific times
+
+(If it is a bug, you are reporting lease specify:)
+
+## Steps to repeat: (Describe how the issue can be repeated by someone who is to work on it)
+1. Step 1
+2. Step 2
+3. ...
+
+## Expected Results:
```

Comment:
+1 on these. 

Basically, if we can shrink advice here (http://www.chiark.greenend.org.uk/~sgtatham/bugs.html) down and get it into a template, that'd be great. This article is perfect.